ChangeLog.md

```diff
@@ -20,6 +20,13 @@ See docs/process.md for more on how version tagging works.
 
 3.1.50 (in development)
 -----------------------
+- Emscripten will now invoke the `clang` or `clang++` when compiling depending
+  only on whether `emcc` or `em++` was run.  Previously it would determine
+  which to run based on the invidifual file extension.  One side effect of this
```

Comment:
How about the other direction - what happens when passing `.cpp` to `emcc` now? Also a warning / a hard error / something else?

Reply:
Nvm, saw answer in the other PR.